datm/datm_datamode_era5_mod.F90

```diff
@@ -265,8 +283,11 @@ subroutine datm_datamode_era5_advance(exportstate, masterproc, logunit, mpicom,
        Faxa_swvdf(n) = Faxa_swdn(n)*Faxa_swvdf(n)
        Faxa_swndf(n) = Faxa_swdn(n)*Faxa_swndf(n)
 
+       !--- TODO: need to understand relationship between shortwave bands and net shortwave rad.
+       !--- currently it is provided directly from ERA5 and the total of the bands are not
+       !--- consistent with the swnet
        !--- swnet: a diagnostic quantity ---
-       Faxa_swnet(n) = Faxa_swndr(n) + Faxa_swvdr(n) + Faxa_swndf(n) + Faxa_swvdf(n)
+       !Faxa_swnet(n) = Faxa_swndr(n) + Faxa_swvdr(n) + Faxa_swndf(n) + Faxa_swvdf(n)
```

Comment:
Could you pease add this as an issue in CDEPS. We should translate the TODOs to issues and then remove the TODOs.

Reply:
Sure. I'll create an issue for it. I have still active ticket about it in ECMWF side and I could also add it to issue.

Reply:
Issue created, https://github.com/ESCOMP/CDEPS/issues/8